doctr/datasets/multithreading.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+
+import multiprocessing as mp
+from multiprocessing.pool import ThreadPool
+from typing import Callable, Any, Optional, Iterable
+
+
+__all__ = ['multithread_exec']
+
+
+def multithread_exec(func: Callable[[Any], Any], seq: Iterable[Any], threads: Optional[int] = None) -> Iterable[Any]:
+    """Download a file accessible via URL with mutiple retries
```

Comment:
I think this is not appropriated to this function

Reply:
Good catch!